src/gui/systray.cpp

```diff
@@ -193,6 +194,37 @@ void Systray::setPauseOnAllFoldersHelper(bool pause)
     }
 }
 
+void Systray::slotForceSyncAllFolders()
+{
+    // For some reason we get the raw pointer from Folder::accountState()
+    // that's why we need a list of raw pointers for the call to contains
+    // later on...
+    const auto accounts = [=] {
+        const auto ptrList = AccountManager::instance()->accounts();
+        auto result = QList<AccountState *>();
+        result.reserve(ptrList.size());
+        std::transform(std::cbegin(ptrList), std::cend(ptrList), std::back_inserter(result), [](const AccountStatePtr &account) {
+            return account.data();
+        });
+        return result;
+    }();
+    FolderMan *folderMan = FolderMan::instance();
+    const auto folders = folderMan->map();
+    for (auto f : folders) {
+        if (accounts.contains(f->accountState())) {
+            if (f->isSyncRunning()) {
+                f->slotTerminateSync();
+                folderMan->scheduleFolder(f);
```

Comment:
I would use scheduleFolderNext instead, so the folder will be the first in the queue.
Does this answer your question https://github.com/nextcloud/desktop/pull/3696#issue-716697469?

Reply:
I am just a bit puzzled. There is functionality for force sync for a *single* account. It is right here:

https://github.com/nextcloud/desktop/blob/a5e45d521349b2cd6347b21a12b5195d90f5d7f2/src/gui/accountsettings.cpp#L1034

So what I need is basically loop over all accounts with that function. If you are proposing to change that code, I guess the above code should also be changed?

As I understand it the function adds all folder to be synced, then adds the root node as first in queue. Probably because there might be changes-to-be-synced under root that make any subsequent sync to subfolders obsolete? (I.e. deleting 
 a subfolder of root making any scheduleFolder for those children go poof). Not sure.

the 
`selectedFolder->slotWipeErrorBlacklist(); // issue #6757
`is completely beyond my comprehension.